src/ralph/admin/filters.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+import re
+from datetime import datetime
+
+from django.contrib.admin import SimpleListFilter
+from django.db.models import Q
+from django.utils.encoding import smart_text
+from django.utils.formats import get_format
+from django.utils.translation import ugettext_lazy as _
+
+
+class BaseCustomFilter(SimpleListFilter):
+
+    def has_output(self):
+        return True
+
+    def lookups(self, request, model_admin):
+        return None
+
+
+class BooleanFilter(BaseCustomFilter):
```

Comment:
couldn't `BooleanFilter` inherit from `ChoicesFilter` and just overwrite `self.choices_list`?